MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -181,7 +181,7 @@ public GraphicsAdapter Adapter
         /// <summary>
         /// Access debugging APIs for the graphics subsystem.
         /// </summary>
-        public GraphicsDebug GraphicsDebug { get { return _graphicsDebug; } set { _graphicsDebug = value; } }
+        public GraphicsDebug GraphicsDebug { get { return _graphicsDebug; } }
```

Comment:
We should add either to these docs or somewhere that this feature only works on some platforms.  Might even call out DX and GL platforms specifically since that is all that is supported.